platforms/hyperledger-besu/configuration/README.md

```diff
@@ -1,49 +1,49 @@
-# Playbooks
+# Hyperledger Besu Configuration
+This configuration folder contains Ansible playbooks and their corresponding roles, which are used to deploy Hyperledger Besu on Managed Kubernetes Cluster(s).
 
-## Getting started
-This folder contains ansible playbooks and their corresponding roles which are used to deploy Hyperledger Besu on Managed Kubernetes Cluster(s).
 
+## Approaches
+There are two approaches to deploy a DLT network using BAF: 
+- Use a dedicated instance to deploy and manage the DLT network. This is recommended for production environments, and requires some setup & manual installation of CLI tools. More info on setting up this dedicated instance is found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/operations/configure_prerequisites.html#ansible-inventory-file).
+- Use the 'BAF Build container' to create a containerized Ansible controller from which to deploy/manage your networks. This is recommended for development instances, as it is an easy way to build the required base environment for BAF deployment. More info can be found [here](https://blockchain-automation-framework.readthedocs.io/en/latest/developer/docker-build.html).
 
-## Prerequisites
+## Installation pre-requisites
+Some tools are required to run the BAF Automation, such as Kubernetes, Git (repository), Vault and more.
```

Comment:
Don't Some tools , rephrase to say BAF requires or makes use of

Reply:
Resolved with commit [5f62fa1](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/1038/commits/5f62fa1c30d332957c60305b7555b9d8466594a9)